pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/defaultvalues/RgpAutomaticProbandDataCreator.java

```diff
@@ -1,9 +1,9 @@
 package org.broadinstitute.dsm.model.defaultvalues;
 
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```

Comment:
I _think_ we avoid wildcard imports where possible.

Reply:
Good spot. My guess is IntelliJ provided this but I missed it.